apps/science/include/ElevatorConfig.h

```diff
@@ -14,27 +14,30 @@
 namespace Elevator {
 
 namespace Internal {
-static Actuator::DCMotor motor(MTR_PWM_2, MTR_DIR_2, false);
+static Actuator::DCMotor motor(MTR_PWM_ELVTR, MTR_DIR_ELVTR, false);
 
-// Felix TODO: ensure that Elevator uses Pololu encoder
-static Encoder::Pololu37D encoder({NC, NC, NC, 0});
+// channela, channelb, index, offset deg, qei encoding
+static Encoder::Pololu37D encoder({ENC_A_LIFT, ENC_B_LIFT, NC, 0});
 
 static PID::PID velPID({1, 0, 0, -1, 1, 0, false, false});
 static PID::PID posPID({1, 0, 0, -1, 1, 0, false, false});
 
-constexpr float MAXDEGPERSEC =
-    std::numeric_limits<float>::infinity();  // TODO: figure out MAXDEGPERSEC of motors (1197.482?)
-constexpr float MAXCURRENT = std::numeric_limits<float>::infinity();  // since no current sensor
+constexpr float MAX_DEG_PRE_SEC = 1197;                                    // 20.9 RAD/s
+constexpr float MAX_CURRENT     = std::numeric_limits<float>::infinity();  // since no current sensor
 
-static Controller::Velocity vel(motor, encoder, std::nullopt, velPID, MAXDEGPERSEC, MAXCURRENT, LIM_SW_3, LIM_SW_4);
-static Controller::Position pos(motor, encoder, std::nullopt, posPID, MAXDEGPERSEC, MAXCURRENT, LIM_SW_3, LIM_SW_4);
-static Controller::OpenLoop open(motor, encoder, std::nullopt, MAXDEGPERSEC, MAXCURRENT, LIM_SW_3, LIM_SW_4);
+static Controller::Velocity vel(motor, encoder, std::nullopt, velPID, MAX_DEG_PRE_SEC, MAX_CURRENT, LIM_SW_ELVTR_DN,
+                                LIM_SW_ELVTR_UP);
+static Controller::Position pos(motor, encoder, std::nullopt, posPID, MAX_DEG_PRE_SEC, MAX_CURRENT, LIM_SW_ELVTR_DN,
+                                LIM_SW_ELVTR_UP);
+static Controller::OpenLoop open(motor, encoder, std::nullopt, MAX_DEG_PRE_SEC, MAX_CURRENT, LIM_SW_ELVTR_DN,
+                                 LIM_SW_ELVTR_UP);
 
-static const Controller::ControlMap lut = {{HWBRIDGE::CONTROL::Mode::Velocity, &vel},
-                                           {HWBRIDGE::CONTROL::Mode::Position, &pos},
-                                           {HWBRIDGE::CONTROL::Mode::OpenLoop, &open}};
+static const LookupTable::LookupTable<HWBRIDGE::CONTROL::Mode, Controller::ActuatorController *> lut = {
+    {HWBRIDGE::CONTROL::Mode::Velocity, &vel},
+    {HWBRIDGE::CONTROL::Mode::Position, &pos},
+    {HWBRIDGE::CONTROL::Mode::OpenLoop, &open}};
```

Comment:
i think you didnt mean to change this

Reply:
uh I don't recall changing this. I think I did a merge with your branch so that might have caused this